README.rst

```diff
@@ -30,13 +30,17 @@
     :target: https://github.com/melexis/sphinx-traceability-extension/issues
     :alt: Contributions welcome
 
-===================================
-Sphinx Traceability plugin - README
-===================================
+==========================
+Sphinx Traceability plugin
+==========================
 
 Sphinx plugin that allows defining documentation items and relations between those items. Can be used
 as a requirements management tool for e.g. ISO26262 projects.
 
+.. contents:: `Contents`
```

Comment:
This makes a nice toc indeed, but the title of TOC is strange. Maybe change it to `== Contents ==` ?

Reply:
This is the TOC title in every file in _Example usage_. Do you want it changed there as well?

Reply:
This is a different beast as it seems at least in markdown. We can check rendered view to see whats going on...